e2e/scripts/st_add_rows.py

```diff
@@ -42,7 +42,7 @@
 
 for test_type in ["coalesce in Py", "coalesce in JS", "clear after addrows"]:
 
-    table_el = st.table(df1)
+    table_el = st.old_table(df1)
```

Comment:
Do we keep this `old_table` API?

Reply:
This will not go into production. I left the old methods there for the "final" testing. Will remove them as soon as we are confident that the new methods cover all test cases that the previous methods did (+ some more 🎉 ).